examples/insurance/src/main/ml-modules/root/custom-modules/custom/enrichAdvantage/main.sjs

```diff
@@ -0,0 +1,78 @@
+/*
+  Copyright 2012-2019 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+const DataHub = require("/data-hub/5/datahub.sjs");
+const datahub = new DataHub();
+// Require module for mapping latitude and longitude from zip codes
+const zipcodeData = require("/custom-modules/utils/zipcodeData.sjs");
+   
+function main(content, options) {
+
+  //let's set our output format, so we know what we're exporting
+  let outputFormat = options.outputFormat ? options.outputFormat.toLowerCase() : datahub.flow.consts.DEFAULT_FORMAT;
+   
+  //grab the 'doc' from the content value space
+  let doc = content.value;
+   
+  // let's just grab the root of the document if its a Document and not a type of Node (ObjectNode or XMLNode)
+  if (doc && (doc instanceof Document || doc instanceof XMLDocument)) {
+    doc = fn.head(doc.root);
+  }
+   
+  //get our instance, default shape of envelope is envelope/instance, else it'll return an empty object/array
```

Comment:
I recommend removing most of these comments, as they don't add any value, they're just explaining what the code is clearly doing. They make the module seem more complicated than it really is.